examples/assistants-azure.ts

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env -S npm run tsn -T
+
+import { AzureOpenAI } from 'openai';
+
+/**
+ * Example of polling for a complete response from an assistant
+ */
+const apiVersion = '2024-02-15-preview';
+const openai = new AzureOpenAI({
+  endpoint:'https://YOUR_AZURE_OPENAI_RESOURCE_NAME.openai.azure.com/',
+  apiKey: 'YOUR_AZURE_API_KEY',
+  apiVersion
+});
+
+
+async function main() {
+  const assistant = await openai.beta.assistants.create({
+    model: 'gpt-4',
+    name: 'Math Tutor',
+    instructions: 'You are a personal math tutor. Write and run code to answer math questions.',
+    // tools = [],
+  });
+
+  let assistantId = assistant.id;
+  console.log('Created Assistant with Id: ' + assistantId);
```

Comment:
nitpick: simplifying:
```suggestion
  console.log('Created Assistant with Id: ' + assistant.id);
```